src/VisualStudio/Core/Def/Implementation/NavigationBar/NavigationBarClient.cs

```diff
@@ -152,7 +152,7 @@ int IVsDropdownBarClient.GetComboTipText(int iCombo, out string pbstrText)
             int selectionIndex;
             var selectedItemPreviewText = string.Empty;
 
-            if (_dropdownBar.GetCurrentSelection(iCombo, out selectionIndex) == VSConstants.S_OK)
+            if (_dropdownBar.GetCurrentSelection(iCombo, out selectionIndex) == VSConstants.S_OK && selectionIndex >= 0)
```

Comment:
The [documentation for the API](https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.textmanager.interop.ivsdropdownbar.getcurrentselection.aspx) isn't clear if -1 is an allowed return or not.

@brettfo: sounds like that's a separate bug. Even if it's not a separate PR, please have a separate git commit and an explanation in that commit why it's there.